app/views/assignments/_form.html.erb

```diff
@@ -61,7 +61,7 @@
 
 
   <div class="actions">
-    <%= form.submit %>
+    <%= form.submit %, class="btn btn-info">
```

Comment:
Please correct the syntax here, this should be 
`<%= form.submit, class:"btn btn-info" %>`
The tests seem to be failing due to this, also check your changes on Gitpod first.